### PR TITLE
Added the possibility to send logs to multiple destinations with HTTP

### PR DESCRIPTION
**Pull request by ajacquemot**
### What does this PR do?

Make sure additional endpoints are parsed for HTTP config and updated the client to sends logs in async 

### Motivation

Send logs to multiple destinations with HTTP

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: pkg/logs/config/config.go [hunk 1]
```diff
@@ -149,13 +150,23 @@ func buildHTTPEndpoints() (*Endpoints, error) {
...
+	}
+	main.UseSSL = useSSL
+
+	var additionals []Endpoint
+	err := coreConfig.Datadog.UnmarshalKey("logs_config.additional_endpoints", &additionals)
+	if err != nil {
+		log.Warnf("Could not parse additional_endpoints for logs: %v", err)
+	}
+	for i := 0; i < len(additionals); i++ {
+		additionals[i].UseSSL = useSSL
```
Comment (blemale) [inline pkg/logs/config/config.go]: np: WDYT of not introduce a local variable for `useSSL` and use explicitly the main configuration for the additional endpoints, i.e.: 
```suggestion
		additionals[i].UseSSL = main.useSSL
```

**Review by blemale — APPROVED** [review]

File: pkg/logs/config/config.go (hunk 1 above)
Comment (ajacquemot) [reply pkg/logs/config/config.go]: 👍

File: pkg/logs/client/http/destination.go [hunk 2]
```diff
@@ -86,9 +89,29 @@ func (d *Destination) Send(payload []byte) error {
 	}
 }
 
-// SendAsync is not implemented for HTTP.
+// SendAsync sends a payload in background.
 func (d *Destination) SendAsync(payload []byte) {
-	return
+	d.once.Do(func() {
+		payloadChan := make(chan []byte, 100)
```
Comment (gaetan-deputier) [inline pkg/logs/client/http/destination.go]: Can we use config.ChanSize maybe?

Comment (ajacquemot) [reply pkg/logs/client/http/destination.go]: 👍

**Pull request merged**